JL.Core/Mining/MiningUtils.cs

```diff
@@ -440,6 +686,76 @@ public static async Task MineToFile(LookupResult lookupResult, string currentTex
         Utils.Logger.Information("Mined {PrimarySpelling}", lookupResult.PrimarySpelling);
     }
 
+    public static async ValueTask<bool[]?> CheckDuplicates(List<LookupResult> lookupResults, string currentText, int currentCharPosition)
```

Comment:
```suggestion
    public static async ValueTask<bool[]?> CheckDuplicates(LookupResult[] lookupResults, string currentText, int currentCharPosition)
```

In the master branch `LookupResults` is now an array instead of a list.